pkg/controller/windowsmachine/windowsmachine_controller.go

```diff
@@ -302,8 +313,10 @@ func (r *ReconcileWindowsMachine) Reconcile(request reconcile.Request) (reconcil
 		}
 
 		if _, present := node.Annotations[nodeconfig.VersionAnnotation]; present {
-			// version annotation doesn't match the current operator version
-			if node.Annotations[nodeconfig.VersionAnnotation] != version.Get() {
+			// If either the version annotation doesn't match the current operator version, or the private key used
+			// to configure the machine is out of date, the machine should be deleted
+			if node.Annotations[nodeconfig.VersionAnnotation] != version.Get() ||
+				node.Annotations[nodeconfig.PubKeyHashAnnotation] != nodeconfig.CreatePubKeyHashAnnotation(r.signer.PublicKey()) {
```

Comment:
This log [message](https://github.com/openshift/windows-machine-config-operator/pull/239/commits/5f997795192e75e5b59b117fdc2692e45722db1e#diff-f6cf75653deff7ed72be94f059acc292d15113ed7c9f9ce0fdc5e842fa71aa72R324) is no longer true and needs to be updated